sound/soc/intel/boards/sof_sdw.c

```diff
@@ -379,6 +379,11 @@ static struct sof_sdw_codec_info codec_info_list[] = {
 		.dai_name = "rt5682-sdw",
 		.init = sof_sdw_rt5682_init,
 	},
+	{
+		.acpi_id = "CnviBt",
```

Comment:
you're killing me.

Reply:
I know, I know, just to make sure this is no-go.

Reply:
This is handled in latest revision, marking as resolved.